transports/tcp/src/lib.rs

```diff
@@ -394,14 +391,15 @@ impl Drop for TcpTransStream {
 
 #[cfg(test)]
 mod tests {
+    extern crate tokio;
+    use self::tokio::runtime::current_thread::Runtime;
```

Comment:
is `self` needed here?

Reply:
I could use tests instead of self but it fails resolve without either.

Reply:
No, I think it's fine!